scripts/update_and_rebuild_libmesh.sh

```diff
@@ -113,13 +113,13 @@ fi
 # with either JOBS if set, or LIBMESH_JOBS.
 LIBMESH_JOBS=${MOOSE_JOBS:-1}
 
-if [ -z "${MOOSE_MAKE}" ]; then
-  make -j ${JOBS:-$LIBMESH_JOBS} && \
-    make install
-else
-  ${MOOSE_MAKE} && \
-    ${MOOSE_MAKE} install
-fi
+#if [ -z "${MOOSE_MAKE}" ]; then
+#  make -j ${JOBS:-$LIBMESH_JOBS} && \
```

Comment:
I'm going to guess the problem with the reason this PR didn't work has nothing at all to do with CIVET...